tests/model_registry/rest_api/conftest.py

```diff
@@ -47,3 +59,47 @@ def updated_model_artifact(
         headers=model_registry_rest_headers,
         data_json=request.param,
     )
+
+
+@pytest.fixture(scope="class")
+def deploy_secure_mysql_and_mr(
+    model_registry_namespace: str,
+    model_registry_db_secret: Secret,
+    model_registry_db_service: Service,
+    model_registry_db_deployment: Deployment,
+) -> Generator[None, None, None]:
+    """
+    Fixture to deploy MySQL with SSL/TLS and a Model Registry configured to use a secure DB connection.
+
+    Parameters:
+        admin_client (object): Kubernetes/OpenShift admin client for resource operations.
+        model_registry_namespace (str): Namespace where resources are deployed.
+        model_registry_db_secret (Secret): Secret resource for MySQL credentials.
+        model_registry_db_service (Service): Service resource for MySQL DB.
+        model_registry_db_deployment (Deployment): Deployment resource for MySQL DB.
+    """
+
+    mysql_template = get_model_registry_deployment_template_dict(
+        secret_name=model_registry_db_secret.name,
+        resource_name=DB_RESOURCES_NAME,
+    )
+
+    ssl_ca_arg = f"--ssl-ca={CA_FILE_PATH}"
```

Comment:
You don't need to save it, you can use it directly below. Same comment for ca_volume_mount, ca_volume and patch.